environments/puppet/hiera.yaml

```diff
@@ -23,16 +56,41 @@ hierarchy:
     - "%{facts.os.name}/%{facts.os.release.major}.yaml"
     - "%{facts.os.name}.yaml"
 
+  - name: EYAML per-hostgroup data
+    lookup_key: eyaml_lookup_key
+    paths:
+    - "hostgroups/%{hostgroup}.eyaml"
+    options:
+      pkcs7_private_key: /var/lib/puppet/keys/private_key.pkcs7.pem
+      pkcs7_public_key: /var/lib/puppet/keys/public_key.pkcs7.pem
+
   - name: Per-hostgroup data - See site.pp for more information
     paths:
-    - "hostgroups/%{::hostgroup}.yaml"
+    - "hostgroups/%{hostgroup}.yaml"
+
+  - name: EYAML General data
+    lookup_key: eyaml_lookup_key
+    paths:
+    - "common.eyaml"
+    options:
+      pkcs7_private_key: /var/lib/puppet/keys/private_key.pkcs7.pem
+      pkcs7_public_key: /var/lib/puppet/keys/public_key.pkcs7.pem
 
   - name: General data
     paths:
     - "default.yaml"
     - "common.yaml"
 
+  - name: EYAML SIMP specific data - Please do not modify
+    lookup_key: eyaml_lookup_key
+    paths:
+    - "simp_config_settings.eyaml"
+    - "scenarios/%{simp_scenario}.eyaml"
```

Comment:
This is a top-scope variable, so [it should have a `::` prefix](https://puppet.com/docs/puppet/latest/hiera_merging.html#interpolate_puppet_variable).

```suggestion
    - "scenarios/%{::simp_scenario}.eyaml"
```